File: src/Problems/Problem_1.hs
-- 233168 - Completed 29.4.2013
module Problems.Problem_1 
    ( problem_1 
    , problem_1'
    ) where

import Data.List (union)

problem_1 = sum [x | x <- [1..999], x `rem` 3 == 0 || x `rem` 5 == 0]

problem_1' = sum $ union [3,6..999] [5,10..995]

File: src/C_Problems/Problem_1.c
// 233168 - Completed 20.8.2013
#include <stdlib.h>
#include <stdio.h>

int main(){
    int sum = 0;
    for (int n = 0; n < 1000; n++){
	sum = (n % 3 == 0 || n % 5 == 0) ? sum + n : sum;
    }
    printf("%d\n",sum);
}
